docker-compose.yml

```diff
@@ -0,0 +1,73 @@
+version: "3"
+
+services:
+  mariadb:
+    image: mariadb:10.3
+    ports:
+      - '3306'
+    environment:
+      MYSQL_ROOT_PASSWORD: password
+      MYSQL_USER: db
+      MYSQL_PASSWORD: db
+      MYSQL_DATABASE: db
+
+  phpfpm:
+    image: itkdev/php7.2-fpm:latest
+    restart: always
+    environment:
+      - PHP_XDEBUG=${PHP_XDEBUG:-0}
+      - PHP_XDEBUG_REMOTE_AUTOSTART=${PHP_XDEBUG_REMOTE_AUTOSTART:-0}
+      - PHP_XDEBUG_REMOTE_HOST=${PHP_XDEBUG_REMOTE_HOST:-0.0.0.0}
+      - PHP_XDEBUG_REMOTE_CONNECT_BACK=${PHP_XDEBUG_REMOTE_CONNECT_BACK:-0}
+      # - PHP_MAIL=1 # Uncomment to enable mailhog
+    depends_on:
+      - mariadb
+    volumes:
+      - .:/app:delegated
+
+  nginx:
+    image: nginx:latest
+    depends_on:
+      - phpfpm
+    ports:
+      - '80'
+    volumes:
+      - ${PWD}/.docker/vhost.conf:/etc/nginx/conf.d/default.conf:ro
+      - ./:/app:delegated
+    labels:
+      - "traefik.frontend.rule=Host:${COMPOSE_PROJECT_NAME}.docker.localhost"
+
+  memcached:
+    image: 'memcached:latest'
+    ports:
+      - '11211'
+    environment:
+      - MEMCACHED_CACHE_SIZE=64
+
+  mailhog:
+    image: mailhog/mailhog
+    restart: always
+    ports:
+      - "1025"
+      - "8025"
+
+  reverse-proxy:
+    image: traefik  # The official Traefik docker image
+    command: --api --docker  # Enables the web UI and tells Traefik to listen to docker
+    ports:
+      - "80"
+      - "8080"
+    volumes:
+      - /var/run/docker.sock:/var/run/docker.sock  # So that Traefik can listen to the Docker events
+
+  # varnish:
```

Comment:
Comment out = Delete
This will also remove the ereolen name from the yml :)

Reply:
Fixed!